pytket/docs/build-docs.sh

```diff
@@ -2,8 +2,6 @@ cp -R pytket-docs-theming/_static .
 cp -R pytket-docs-theming/quantinuum-sphinx .
 cp pytket-docs-theming/conf.py .
 
-echo "coverage_ignore_functions.extend([\"get_job_shot_num\", \"get_rng_num\", \"set_rng_bound\", \"set_rng_index\", \"set_rng_seed\"])" >> conf.py
-
```

Comment:
Now that we are removing the `_tket` prefix is it possible to remove these lines from the `build-docs.sh` script?

```shell
if [[ "$OSTYPE" == "darwin"* ]]; then
    find build/ -type f -name "*.html" | xargs sed -e 's/pytket._tket/pytket/g' -i ""
    sed -i '' 's/pytket._tket/pytket/g' build/searchindex.js
else
    find build/ -type f -name "*.html" | xargs sed -i 's/pytket._tket/pytket/g'
    sed -i 's/pytket._tket/pytket/g' build/searchindex.js
fi
```

The purpose of this script is to remove all `_tket` links which show up in the html after the sphinx build runs.

Reply:
I tried removing it locally and everything seemed to build perfectly fine for me. To clarify, any stubs generated through nanobind will still include and reference `_tket` so we can't avoid mentioning it at all on the docs, but they still seem to link fine without this fix